backend/coins/btc/account_test.go

```diff
@@ -110,7 +111,7 @@ func TestAccount(t *testing.T) {
 	account := mockAccount(t, nil)
 	require.False(t, account.Synced())
 	require.NoError(t, account.Initialize())
-	require.True(t, account.Synced())
+	require.Eventually(t, func() bool { return account.Synced() }, time.Minute, time.Second)
```

Comment:
TIL Eventually :exploding_head: 

Do you know how long it takes? Maybe a shorter interval could make sense.

Reply:
I think it barely needs to wait, I can definitely try to lower the value and see what happens :)